gradle/gradle-bintray.gradle

```diff
@@ -52,7 +52,7 @@ publishing {
 bintray {
     user = mapboxBintrayUser
     key = mapboxBintrayApiKey
-    publications = ['MapboxJavaSDKPublication']
```

Comment:
Could you perform a dry-run and verify whether javadocs and sources are correctly included?

Reply:
Was able to successfully publish a dry-run `4.10.0-SNAPSHOT` to Bintray from my local machine

![Screen Shot 2019-10-04 at 1 59 18 PM](https://user-images.githubusercontent.com/4394910/66239658-54548000-e6af-11e9-8bbd-7675c796221e.png)
![Screen Shot 2019-10-04 at 1 59 21 PM](https://user-images.githubusercontent.com/4394910/66239660-5585ad00-e6af-11e9-810c-9e010b107995.png)
![Screen Shot 2019-10-04 at 1 58 23 PM](https://user-images.githubusercontent.com/4394910/66239669-5c142480-e6af-11e9-938e-bae72fa89bbf.png)
![Screen Shot 2019-10-04 at 1 58 10 PM](https://user-images.githubusercontent.com/4394910/66239670-5cacbb00-e6af-11e9-8c4d-7daeee6a4a2d.png)
![Screen Shot 2019-10-04 at 1 58 03 PM](https://user-images.githubusercontent.com/4394910/66239671-5cacbb00-e6af-11e9-926d-0a013fecc60b.png)
![Screen Shot 2019-10-04 at 2 01 25 PM](https://user-images.githubusercontent.com/4394910/66239729-80700100-e6af-11e9-86b8-0f4efe31e324.png)

Reply:
I also made a test tag that targeted this pr's branch, to see if the `circle.yml` changes in this pr still successfully got files from CircleCI to Bintray. Looks like a success.  
![Screen Shot 2019-10-04 at 2 15 27 PM](https://user-images.githubusercontent.com/4394910/66240474-97afee00-e6b1-11e9-9c88-b735d904084c.png)
![Screen Shot 2019-10-04 at 2 16 32 PM](https://user-images.githubusercontent.com/4394910/66240477-98e11b00-e6b1-11e9-80aa-22b1d1d20fa8.png)


![Screen Shot 2019-10-04 at 2 17 43 PM](https://user-images.githubusercontent.com/4394910/66240629-08efa100-e6b2-11e9-96f6-77777728267a.png)
![Screen Shot 2019-10-04 at 2 17 41 PM](https://user-images.githubusercontent.com/4394910/66240630-08efa100-e6b2-11e9-8b23-6d9b727cd4ff.png)
![Screen Shot 2019-10-04 at 2 17 38 PM](https://user-images.githubusercontent.com/4394910/66240632-08efa100-e6b2-11e9-9b6f-1705087a530d.png)
![Screen Shot 2019-10-04 at 2 17 27 PM](https://user-images.githubusercontent.com/4394910/66240633-08efa100-e6b2-11e9-8cd2-f5f11a64a850.png)